sphinx/writers/latex.py

```diff
@@ -275,6 +275,22 @@ def escape_abbr(text):
     return re.sub('\.(?=\s|$)', '.\\@', text)
 
 
+def rstdim_to_latexdim(width_str):
+    """Convert `width_str` with rst length to LaTeX length."""
+    match = re.match('^([0-9.]+)\s*(\S*)$', width_str)
```

Comment:
This is copied from docutils'.
https://sourceforge.net/p/docutils/code/HEAD/tree/trunk/docutils/docutils/parsers/rst/directives/__init__.py#l233

But this is not strict as you said. So I'll revert this.